### ISSUE TITLE
ARM image

### ISSUE THREAD

**Issue by nemchik** [manbearwiz/youtube-dl-server#23]
Would you be able to build your image for ARM architecture as well?

**Comment by manbearwiz** [comment]
I don't have an arm docker server right now. If someone wants to make a PR for this I would accept it. It probably just means changing the base image in the dockerfile.

**Comment by maximilianwollnik** [comment]
I have tested your latest version but unfortunately it does not start on a RPI4 (Raspbian Buster with Docker 19.03.2) . Then I have checked the docker-image **python:alpine** again and actually this image works on my pi. So I have just cloned this repo and I have just built this image again and the container started successfully.

So not sure what is the best approach. Just rebuild the image or maybe just change the base-image to a fix version like **python:alpine:3.9**. For now I can ensure, that this version works on my pi.

**Comment by manbearwiz** [comment]
I just built a new image if you want to check it out. Should be based on python@3.7.4 and alpine@3.10

**Comment by maximilianwollnik** [comment]
Unfortunately it is a matter of your build-environment. It seems to be crucial, that this image must be built on all plattforms you want to support. If you can't provide this, then maybe this issue should lead to a documentation change with a hint to rebuild the image on the target plattform.

**Comment by MHCraftbeer** [comment]
@maximilianwollnik could you please describe how you got the container started on arm? How did you build the image?

**Comment by MHCraftbeer** [comment]
Well, I did it on my own.
Here is the solution for my Odroid-HC2 (armhf) openmediavault 5 (Debian 10 Buster) server:
`git clone https://github.com/manbearwiz/youtube-dl-server`
`cd youtube-dl-server`
`docker build . -t local-youtube -f Dockerfile`
`docker run -d --net="host" --name youtube-dl -v /srv/dev-disk-by-label-WD8TB/common/Downloads:/youtube-dl --restart unless-stopped local-youtube:latest`

I hope it helps fellow users.

**Comment by maximilianwollnik** [comment]
@MHCraftbeer: yeah, this is what I have done by myself. Therefore I think this issue can be closed with a documentation update
